package.json

```diff
@@ -31,10 +31,10 @@
   },
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
-    "@edx/frontend-component-footer": "^11.5.0",
-    "@edx/frontend-component-header": "^3.4.0",
-    "@edx/frontend-platform": "^3.1.1",
-    "@edx/paragon": "^20.0.0",
+    "@edx/frontend-component-footer": "11.7.3",
+    "@edx/frontend-component-header": "3.7.3",
+    "@edx/frontend-platform": "4.1.0",
+    "@edx/paragon": "20.28.5",
```

Comment:
As long as we're in here it would be better to update from using the redirect `@edx/paragon` to `@openedx/paragon` (and the other packages ,too)

Reply:
To be clear, I think you should make this change here and any place you see it moving forward, but I do not expect you to go back to merged PRs to make this change.